File: app/src/main/java/com/might/dwan/cashcalendar/ui/screens/settings/core/SettingsContractor.java
package com.might.dwan.cashcalendar.ui.screens.settings.core;

import com.might.dwan.cashcalendar.archs.presenters.BaseIPresenter;

public interface SettingsContractor {

    interface IPresenter extends BaseIPresenter {

        void clickNotifications();

    }

    interface IView {

        void goNotificationsSetting();
    }
}

File: app/src/main/java/com/might/dwan/cashcalendar/ui/screens/settings/core/SettingsContractor.kt
package com.might.dwan.cashcalendar.ui.screens.settings.core

import com.might.dwan.cashcalendar.archs.presenters.BaseIPresenter

interface SettingsContractor {

    interface IPresenter : BaseIPresenter {

        fun clickNotifications()

    }

    interface IView {

        fun goNotificationsSetting()
    }
}
